website/docs/r/downtime.html.markdown

```diff
@@ -23,6 +23,12 @@ resource "datadog_downtime" "foo" {
     type   = "days"
     period = 1
   }
+
+  # Datadog API will reject dates in the past so let's ignore `start` and `end`
+  # arguments during lifecycle
+  lifecycle {
+    ignore_changes = ["start", "end"]
```

Comment:
This is likely attached to my initial question around the diffsuppression but not totally clear to me why this needs suppressing?

Reply:
This is because Datadog API doesn't accept dates in the past at the moment you `POST` your request. If you don't ignore `start` after the first `apply`, terraform always tries to `POST` it making the request fail.

Reply:
Discussed this offline. The easiest scenario for when one wants to update/extend a downtime would be to comment out this `ignore` section, apply a new time, and re place the `ignore` section. 

Maybe it'd be helpful to update the comment above with:

```
  # Datadog API will reject dates in the past so let's ignore `start` and `end`
  # arguments during lifecycle. To update or extend an existing downtime, temporarily
  # remove the `ignore` section, apply timestamp changes, and re-apply the `ignore` section.
```
or something similar?